examples/example-predeployed-connector/src/starknet-provider.tsx

```
@@ -0,0 +1,41 @@
+} from "@dojoengine/predeployed-connector";
+
+export default function StarknetProvider({ children }: PropsWithChildren) {
+    const [retries, setRetries] = useState<number>(0);
+    const [connectors, setConnectors] = useState<
+        PredeployedAccountsConnector[]
+    >([]);
+    const provider = jsonRpcProvider({
+        rpc: () => ({ nodeUrl: dojoConfig.rpcUrl as string }),
+    });
+
+    useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: retries

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)